### PR TITLE
Aruha 2785 Allow to reconfigure default retention time and values to put in retention time

### PR DESCRIPTION
**Pull request by antban**
Aruha 2785 Allow to reconfigure default retention time and values to put in retention time

### CODE REVIEW

File: client/Pages/EventTypeCreate/View.elm [hunk 1]
```diff
@@ -330,7 +330,7 @@ viewForm model setup =
                     Help.options
                     Optional
                     Enabled
-                    [ "1", "2", "3", "4" ]
+                    (String.split " " model.userStore.user.settings.retentionTimeDaysValues)
```
Comment (SergKam) [inline client/Pages/EventTypeCreate/View.elm]: It is better to move it to line 180 and use just the resulting variable here.
Plus I would use pipe here 
```
retentionTimeDaysValues =  model.userStore.user.settings.retentionTimeDaysValues |> String.split " "
```

**Comment by antban** [comment]
:+1:

File: client/Pages/EventTypeCreate/View.elm (hunk 1 above)
Comment (antban) [reply client/Pages/EventTypeCreate/View.elm]: @SergKam https://github.com/zalando-nakadi/nakadi-ui/pull/101/commits/641ce62d79454177975b87f6cdd0b34e4a67c8c4

**Comment by adyach** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**